functional-test/src/test/groovy/org/rundeck/util/gui/pages/jobs/JobTabsSpec.groovy

```diff
@@ -0,0 +1,109 @@
+package org.rundeck.util.gui.pages.jobs
+
+import org.rundeck.util.annotations.SeleniumCoreTest
+import org.rundeck.util.container.SeleniumBase
+import org.rundeck.util.gui.pages.execution.ExecutionShowPage
+import org.rundeck.util.gui.pages.execution.HtmlRenderedOutputPage
+import org.rundeck.util.gui.pages.login.LoginPage
+
+@SeleniumCoreTest
+class JobTabsSpec extends SeleniumBase {
+
+    static final String PROJECT_NAME = 'job_nodes_tab'
+
+    def setupSpec() {
+        startEnvironment()
+    }
+
+    void setup() {
+        def loginPage = go(LoginPage)
+        loginPage.login(TEST_USER, TEST_PASS)
+        setupProject(PROJECT_NAME)
+    }
+
+    def cleanup() {
+        deleteProject(PROJECT_NAME)
+    }
+
+    void "job nodes tab"() {
+        when:
+        final String option = 'nodes'
+        def jobShowPage = page(JobShowPage)
+        def executionShowPage = page(ExecutionShowPage)
+        then:
+        def jobCreatePage = go(JobCreatePage, PROJECT_NAME)
+        jobCreatePage.jobNameInput.sendKeys("test-output-tab")
+        jobCreatePage.tab(JobTab.WORKFLOW).click()
+        jobCreatePage.addSimpleCommandStep('echo "hello world"', 0)
+        jobCreatePage.createJobButton.click()
+        jobShowPage.jobActionDropdownButton.click()
+        jobShowPage.waitForElementToBeClickable(jobShowPage.editJobLink)
+        jobShowPage.editJobLink.click()
+        jobCreatePage.tab(JobTab.OTHER).click()
+        jobCreatePage.defaultTabNodes.click()
+        jobCreatePage.updateBtn.click()
+        jobShowPage.runJobBtn.click()
+        def logOutputSwitch = executionShowPage.logOutputSwitch
+        def nodeViewContainer = executionShowPage.nodeFlowState
+        expect:
+        currentUrl.endsWith(option)
+        logOutputSwitch.getAttribute("style") == ""
```

Comment:
what is this supposed to do?

Reply:
this switch don't show if the job output component is rendered to a view, so its a double check.